src/game/models/monsters.js

```diff
@@ -500,7 +500,7 @@ t . Fire Emblem
     prizes: {
       exp: 110,
       items: ['Molten Feather'],
-      equips: ['molten_gauntlets', 'flame'],
+      equips: ['flame', 'magma', 'lava', { id: 'molten_gauntlets', weight: 2 }],
```

Comment:
I find this a bit odd, because some entries are pure string and other entries are objects. In a first look, I don't know what are the weights of `flame`, `magma` and `lava`.